src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailai-image-id']!;
```

Comment:
**Question:** I don't understand why you use the non-null assertion here. Wouldn't it be easier to read the code if we were to check and raise an explicit exception?

```ts
if (!input.dataset.wagtailaiImageId) {
    throw new Error("wagtailai-image-id is empty");
}
```